src/yb/tablet/tablet_bootstrap.cc

```diff
@@ -961,23 +981,11 @@ class TabletBootstrap {
     // Dump the replay state, this will log the pending replicates, which might be useful for
     // debugging.
     vector<string> state_dump;
-    replay_state_->DumpReplayStateToStrings(&state_dump);
     constexpr int kMaxLinesToDump = 1000;
     static_assert(kMaxLinesToDump % 2 == 0, "Expected kMaxLinesToDump to be even");
-    if (state_dump.size() <= kMaxLinesToDump) {
-      for (const string& line : state_dump) {
-        LOG_WITH_PREFIX(INFO) << line;
-      }
-    } else {
-      int i = 0;
-      for (const string& line : state_dump) {
-        LOG_WITH_PREFIX(INFO) << line;
-        if (++i >= kMaxLinesToDump / 2) break;
-      }
-      LOG_WITH_PREFIX(INFO) << "(" << state_dump.size() - kMaxLinesToDump << " lines skipped)";
-      for (i = state_dump.size() - kMaxLinesToDump / 2; i < state_dump.size(); ++i) {
-        LOG_WITH_PREFIX(INFO) << state_dump[i];
-      }
+    replay_state_->DumpReplayStateToStrings(&state_dump, kMaxLinesToDump);
```

Comment:
Maybe replace constant name with `kHalfMaxLinesToDump = 500` and remove the assertion.

Reply:
I removed the assertion.
Having the kMaxLinesToDump would give readers clear idea what's the total number of lines.